wirecard/oxid-ee
# WDPD-21 Fraud Protection

### This PR

* Refactor 'Additional info'
* add Device fingerprint
* add some helper functions

### How to test

* on the admin page `Wirecard Configuration`-`Payment Methods` - `Wirecard Paypal` try any combination of `Shopping Basket`, `Additional Info` and `Descriptor`
* make a payment with  `Wirecard Paypal` see if redirected correctly with and without separate billing address

### Jira Links
https://jira.parkside.at/browse/WDPD-21
https://jira.parkside.at/browse/WDPD-94
